platform/podjs/README.md

```diff
@@ -20,21 +20,48 @@ feature. While we could support the creation of a container and a
 mechanism for loading features in `pod.js` without too much trouble,
 it is currently not implemented.
 
-The `polyOut` and `polyNav` interfaces are not implemented at the
-moment. The former because we don't use it yet, the latter because it
-is currently not part of the `Pod` interface, which we aim to rectify
-soon.
-
 ## Building
 
-Simply run `build.js` in the repository root.
+Simply run `./build.js` in the repository root.
+
+## How to set up a feature to build with podjs
+
+Follow these instructions to do so:
+
+1.  You need to point to `podjs` from the `polypoly-eu` repository and
+    add it as a "development dependency" on your project by running:
+
+    `$npm i <path-that-points-to-platform/podjs --D`
+
+    So, you end up having on your `package.json` the following:
+
+    ```json
+       "devDependencies": {
+          "@polypoly-eu/podjs": "file: <path-that-points-to-platform/podjs>",
+          ...
+       },
+
+    ```
+
+2.  The main step is to copy `pod.js` in your project. For this, please, check the `example` feature as guideline. You should be able to do it with `rollup` tooling (same way as in `example`) or any other script mechanism you desire.
+
+3.  Make sure your `manifest.json` is exposed as `window.manifestData`.
+    So, if your feature is written in `JavaScript` you should require the following in your main index file:
+
+```js
+import manifestData from "<path-to-your-manifest>/manifest.json";
+window.manifestData = manifestData;
+```
+
+Accordingly in any other language.
+
+4.  To verify that everything is in place, you can confirm it with the following steps:
 
-## Usage
+    -   Open your feature's `HTML` file in your browser, `URL`s of your local file should be supported without the need to run an `HTTP server`.
+    -   If you haven't forgotten 'step 3', you should have the correct navigation bar color and the (localized) feature name you have set.
+    -   You can navigate backwards within the feature via the native back button functionality of the browser.
 
-You will need to use this module for creating a polyPod-compatible
-feature. Follow these instructions:
+5.  If everything works, you are ready to start enjoying your new polyPod feature! :)
+    Congrats!
 
-1. Add _podjs_ as a development dependency to your feature.
-2. Copy or link `node_modules/@polypoly-eu/podjs/dist/pod.js`.
-3. Include `pod.js` from your feature's main document - before the
-   feature's own code.
+        > As other `polyPod` platforms will simply ignore the local `pod.js` file and deliver their own version of it, you can leave the `pod.js` file in the feature distribution without any implication.
```

Comment:
Not entirely clear what you mean here. Usual nits apply: backticks for polyPod, and the "comment" format.

Reply:
Hm, if you don't get it, maybe it could be described better. It's basically this:

Even if a feature **ships** with a `pod.js` file, the polyPod will basically pretend that file doesn't exist and deliver their own `pod.js` file instead. This is probably confusing because it's kinda unfortunate naming: Even features that do **not** use _podjs_ include a file called `pod.js`.

I think that's my fault :D Not sure how I'd describe it better here, seems good enough for now.